### PR TITLE
Add debian packing + CI config + readme

### PR DESCRIPTION
**Pull request by xhaggi**
see title

### CODE REVIEW

**Comment by xhaggi** [comment]
@notspiff let me know if you want CI integration with travis or appveyor. If no I'll drop the commits.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Linux
+
+The following instructions assume you will have built Kodi already in the `kodi-build` directory 
+suggested by the README.
+
+1. `git clone https://github.com/xbmc/xbmc.git`
+2. `git clone https://github.com/notspiff/vfs.rar.git`
+3. `cd vfs.rar && mkdir build && cd build`
+4. `cmake -DADDONS_TO_BUILD=vfs.rar -DADDON_SRC_PREFIX=../.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=../../xbmc/kodi-build/addons -DPACKAGE_ZIP=1 ../../xbmc/cmake/addons`
```
Comment (wsnipex) [inline README.md]: we really have to stop using those instructions. At best, they are only useful for devs.
Normal users should never build like this and only get confused.

Comment (xhaggi) [inline README.md]: Sure this introduction is only for development usage

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: RAR VFS for Kodi
+ RAR VFS for Kodi
+
+Package: kodi-vfs-rar-dbg
+Section: debug
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: debug symbols for RAR VFS for Kodi
+debug symbols for RAR VFS for Kodi
```
Comment (wsnipex) [inline debian/control]: space missing at start of line

Comment (xhaggi) [inline debian/control]: Thx, updated.

**Comment by notspiff** [comment]
ci is good. do not use my work ident as maintainer id, use https://github.com/notspiff/visualization.projectm/blob/master/debian/control#L5 please

**Comment by xhaggi** [comment]
@notspiff sure, updated + force-pushed.

**Pull request merged**